src/main/java/com/targetindia/DigitizeTimeTable/service/AdminService.java

```diff
@@ -0,0 +1,163 @@
+package com.targetindia.DigitizeTimeTable.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.targetindia.DigitizeTimeTable.DbSettings;
+import com.targetindia.DigitizeTimeTable.model.ClassInfo;
+import com.targetindia.DigitizeTimeTable.repository.AdminDao;
+import com.targetindia.DigitizeTimeTable.model.Instructor;
+import com.targetindia.DigitizeTimeTable.model.Course;
+import lombok.SneakyThrows;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.sql.ResultSet;
+import java.util.*;
+
+
+@Service
+public class AdminService {
+
+    @Autowired
+    DbSettings dbSettings;
+
+    @Autowired
+    AdminDao dao;
+
+
+
+    public String updateInstructor(Instructor instr){
+        return dao.updateInstructor(instr);
+    }
+    public String updateInstructor(int id,String name,long contact){
+        return dao.updateInstructor(id,name,contact);
+    }
+    @SneakyThrows
+    public String getAllInstructors() {
```

Comment:
public Optional<String> getAllInstructors() {
        try {
            ResultSet resultset = dao.findAllInstructors();
            List<Instructor> instructors = new ArrayList<>();
            while (resultset.next()) {
                int id = resultset.getInt("instructor_id");
                String name = resultset.getString("instructor_name");
                String contact = resultset.getString("contact");
                Instructor instructor = new Instructor(id,name, contact);
                instructors.add(instructor);
            }
            if(instructors.isEmpty()){
                return Optional.empty();
            }
            ObjectMapper objectMapper = new ObjectMapper();
            String jacksonData = objectMapper.writeValueAsString(instructors);
            return Optional.of(jacksonData);
        } catch (Exception e) {
            // handle the exception, e.g. logging
            return Optional.empty();
        }
    }